### ISSUE TITLE
chaman amélio : Arme du Maelstrom

### ISSUE THREAD

**Issue by yannled** [Athissa/Tracker#395]
**Describe the issue you're having**:
Nos attaques automatiques et furie-des-vents sont censés nous donner 5 charge de Maelstrom, mais nous n'avons absolument pas le compte car nos attaques automatiques n'en donne pas.
Je précise que la classe est du coup injouable, et je pèse mes mots. Elle passe de normalement un bon dps à un très mauvais.

**Explain how you expect it work**:
Nos attaques automatiques devraient nous générer du Maelstrom

**Steps to reproduce the problem**:

1. attaquer un mobs sans utiliser de sorts.
2.  voir si le Maelstrom monte.
3. 

**Links to Wowhead, YouTube, etc**:
http://fr.wowhead.com/spell=187890/arme-du-maelstrom

**Comment by GeneralissimeDarklinks** [comment]
À passer en critique car j'ai tester moi même le problème et il y'a 0 regeneration de maelstrom

**Comment by yannled** [comment]
Effectivement, une bug qui bloque totalement une spécialisation devrait être critique et non pas normal. je ne suis de loin pas à me plaindre d'un sort bug mais quand c'est un spécialisation c'est plus problématique.
